### PR TITLE
Org Based and Package Flows

### PR DESCRIPTION
**Pull request by jarenyach**

### CODE REVIEW

**Comment by jarenyach** [comment]
@dcarroll I did a PR from your branch into this branch to get your changes.

File: orgbased.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+- sfdx force:source:push -u ciorg
+  #Run Unit Tests on Scratch Org
+- sfdx force:apex:test:run -u ciorg --wait 10 --resultformat tap --codecoverage --testlevel $TESTLEVEL
+  #Delete Scratch Org
+- sfdx force:org:delete -u ciorg -p
+  #Convert DX to metadata format into new folder based on DEPLOYDIR variable
+- sfdx force:source:convert -r $ROOTDIR -d $DEPLOYDIR
+  #Authorize Target Deployment Org
+- sfdx force:auth:jwt:grant --clientid $UATCONSUMERKEY --jwtkeyfile assets/server.key --username $UATUSERNAME -a UAT
+  #Deploy Target Deployment Org
```
Comment (dcarroll) [inline orgbased.yml]: Deploy and run tests using the deploy command.

**Comment by jarenyach** [comment]
@dcarroll The package based flow is all set for Travis.  Please review.

**Comment by jarenyach** [comment]
Closing this PR since new repos were created.

**Pull request closed without merging**